docs/readme.md

```diff
@@ -22,3 +22,47 @@ footer: MIT Licensed | Copyright © 2018 - Jamiepine
 ::: warning NOTE
 Pulse is still in development, some features are not working yet. In this document they're marked as "coming soon".
 :::
+
+## Install
+
+```
+npm i pulse-framework --save
+```
+
+# Pulse
+
+Pulse is an application logic library for reactive Javascript frameworks with support for VueJS, React and React Native. Lightweight, modular and powerful, but most importantly easy to understand.
+
+## Features
+
+- ⚙️ Modular structure using "collections"
+- ⚡ Cached data & filters with dependency based regeneration
+- ✨ Automatic data normalization
+- 🔒 Model based data validation
+- 🕰️ History tracking with smart undo functions
+- 🔮 Create data relations between collections
+- 🤓 Database style functions
+- 💎 SSOT architecture (single source of truth)
+- 📕 Error logging & snapshot bug reporting
+- 🔧 Wappers for helpers, utilities and service workers
```

Comment:
Can this actually get changed to `Wrappers` please.
Based on: #27

Reply:
Fixed typo